server/authz/policy_test.go

```diff
@@ -500,6 +500,101 @@ func TestAuthorizeSoftwareInventory(t *testing.T) {
 	})
 }
 
```

Comment:
Not sure why these changes are showing on this PR. No need to review this file again. They are the same as what has previously [merged](https://github.com/fleetdm/fleet/pull/18513).